eng/common/templates/jobs/source-build.yml

```diff
@@ -21,6 +21,8 @@ parameters:
   # one job runs on 'defaultManagedPlatform'.
   platforms: []
 
+  displayName: 'Source-Build'
```

Comment:
Nit: from my perspective `parameters` are inputs, i.e. something a user or a YAML calling in the template is providing. Static values are rather `variables`.

Reply:
Jobs templates  unfortunately don't support variables so parameters is the only option aside from duplicating these static values.